airbyte-integrations/connectors/source-s3/source_s3/utils.py

```diff
@@ -50,5 +50,5 @@ def multiprocess_queuer(func: Callable, queue: mp.Queue, *args: Any, **kwargs: A
     queue.put(dill.loads(func)(*args, **kwargs))
 
 
-def get_value_or_json_if_empty_string(options: str) -> str:
-    return options.strip() or "{}"
+def get_value_or_json_if_empty_string(options: str = None) -> str:
+    return options.strip() if options else "{}"
```

Comment:
👍 does what it says!  Replace null options with empty object. 
<img width="684" alt="Screenshot 2023-03-02 at 2 15 23 PM" src="https://user-images.githubusercontent.com/303226/222571559-02cda47e-48d6-4154-8b5f-94fb010100a6.png">

Reply:
The options could contain None or "None", which will cause issues is the future. Leaving it as it is.